packages/store/src/patterns/encodeKey.js

```diff
@@ -85,17 +85,35 @@ const dbEntryKeyToNumber = k => {
   return result;
 };
 
-// BigInts are encoded as keys as follows:
-//   `${prefix}${length}:${encodedNumber}`
+// Positive BigInts are encoded as keys as follows:
+//   `p${lenlen}:${length}:${encodedNumber}`
 // Where:
 //
-//   ${prefix} is either 'n' or 'p' according to whether the BigInt is negative
-//      or positive ('n' is less than 'p', so negative BigInts will sort below
-//      positive ones)
+//   Where the 'p' indicates "positive". Below we explain the current encoding
+//      of negative bigints, which begin with 'n'. 'n' is less than 'p',
+//      so negative BigInts will sort below positive ones.
 //
 //   ${encodedNumber} is the value of the BigInt itself, encoded as a decimal
 //      number.  Positive BigInts use their normal decimal representation (i.e.,
-//      what is returned when you call `toString()` on a BigInt).  Negative
+//      what is returned when you call `toString()` on a BigInt).
+//
+//   ${length} is the decimal representation of the width (i.e., the count of
+//      digits) of the BigInt.
+//
+//   ${lenlen} is the length of the length, which may only be a single digit.
+//      This limits is to bigints smaller that 10 ** 10 ** 10, which is
+//      beyond what we expect any engine to support anyway. (At some point,
+//      we may well impose an implemention limit much smaller than this.)
```

Comment:
Makes me realize that we allow longer negative integers than positive ones. `lenlen` max value is `9` where we have a padding of the length to 10 digits for negative numbers. Since `0` is unused for `lenlen`, maybe a quick fix is to encode `${lenlen-1}` ?

Reply:
Also means the negative digit `length` description is wrong, and you can't produce bingints with `11 ** 10` or more.

Reply:
Done!

Reply:
> Also means the negative digit length description is wrong, and you can't produce negative bingints with 11 ** 10 or more digits.

Could you suggest a correct description? Thanks.